### PR TITLE
Sitrep for supernova

### PR DESCRIPTION
**Pull request by rrobgill**
Addressing Issue #1085 detonation of a Nova Bomb now results in sitrep
to all players that the system is visible.

Signed-off-by: Rob Gill <rrobgill@protonmail.com>

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -7020,6 +7020,12 @@ On %planet%: The colony has reached sufficient size and [[metertype METER_HAPPIN
 SITREP_POP_THRESHOLD_LABEL
 Colonizing Threshold
 
+SITREP_NOVA_DETECTED
+Astronomers have observed a new supernova, destroying everything in that region of %system%!
```
[the remark this answers is not in the document]
Comment (rrobgill) [reply default/stringtables/en.txt]: The star has already gone so the system name is always going to be 'Deep Space'.

I think "in that region of Deep Space" reads better than "in the region of Deep Space".

[the remark this answers is not in the document]
Comment (TheSilentOne1) [reply default/stringtables/en.txt]: "...and turning a once thriving star system into %system%." ?

[the remark this answers is not in the document]
Comment (geoffthemedio) [reply default/stringtables/en.txt]: "once thriving" assumes too much about the reader's opinion of the former contents of the system, I think...

**Comment by geoffthemedio** [comment]
NovaDoubleSitreps
[NovaTest.zip](https://github.com/freeorion/freeorion/files/4742121/NovaTest.zip)
This works, but if there are two nova bombs in a system that are activated on the same turn, then there are two sitrep messages produced. It should probably be just one per system per turn...

**Comment by geoffthemedio** [comment]
Now it generates a single sitrep, but on the turn after the nova has happened.

**Pull request merged**